### PR TITLE
New homepage layouts/copy [FOR REVIEW]

### PR DESCRIPTION
**Pull request by thisisdano**
**A pair with https://github.com/18F/cg-style/pull/237 — they need to update together for everything to work properly**

OK, so I was working from this PR (https://github.com/18F/cg-site/pull/656), trying to work with the new copy and bring the homepage up to current style standards.

In doing so, I updated the style/size of some of the copy elements, which affected their length and presentation. Because I am a dumb idiot who can't keep himself from meddling, I rewrote some of the copy to fit with the new size/flow. (I am truly sorry, but I honestly cannot help myself — I think it's a compulsion.)

Anyhow, I've cleaned up the styles and removed the obvious layout errors. 

Copywise — I hope you can see my meddling as a good-hearted attempt to be helpful, but I understand if you see it otherwise.

cg-homepage-changes

### CODE REVIEW

**Review by jameshupp — APPROVED** [review]
Looks good!

If I read [cg-style pull 237](https://github.com/18F/cg-style/pull/237/files) correctly, looks like the changes here are generic enough that we can reuse them even if we have to move things between sections? (Trying to think of a future in which we decide the content should be reordered again.)

**Comment by jameshupp** [comment]
@thisisdano Can this be pulled in right away, or do you need to do cg-style 237 first? What's the better hygiene?

**Comment by thisisdano** [comment]
It really needs the cg-style PR to go through first — I'll push to get that merged asap

**Comment by thisisdano** [comment]
It's a bit too complicated to sync site and style, imo

**Comment by thisisdano** [comment]
In terms of generic-ness and flexibility, it _should_ be a bit flexible but I'm not 100% sure that reordering or recoding wouldn't have unpredictable consequences.

**Comment by thisisdano** [comment]
And... wrt `cg-style` — we can get the changes merged soon (I think) but they won't actually be _live_ until whenever marco does the next cg-style release (usually wednesday) — so it would make sense to hold off until we know the new styles are available.

**Comment by jameshupp** [comment]
How about this merge? Not into `master` but into `homepage_new_copy`? Still better to hold off, or should I merge it in and state that `cg-style` is a dependency before merging #656?

**Comment by thisisdano** [comment]
This "sub-merge" should be fine — but it just won't totally look right locally unless you are running `dw-site-style` in `cg-style`

File: layouts/index.html
Comment (msecret) [inline layouts/index.html]: `b-blue bt2px`, look like utility classes, shouldn't they be removed?

**Review by msecret — CHANGES_REQUESTED** [review]

File: layouts/index.html
Comment (thisisdano) [reply layouts/index.html]: 'tis true — I built this without thinking about that

Comment (thisisdano) [reply layouts/index.html]: looks like I'll have to add some new classes to the ol' master

**Comment by thisisdano** [comment]
Fixed utility class stuff — but now there's a new PR needed in cg-style. 
https://github.com/18F/cg-style/pull/240

**Comment by jameshupp** [comment]
Not totally sure if this "sub-merge" as Dan called it is ready to go based on @msecret's comment and @thisisdano's subsequent changes but if it is feel free to go ahead any time (or get someone else to do so, for appropriate merging's sake)! I'm going to be out for a few days so can't handle it myself for a little bit.

**Review by adborden — APPROVED** [review]

**Pull request merged**
